api/tests/integration/infrastructure/repositories/membership-repository_test.js

```diff
@@ -505,51 +542,28 @@ describe('Integration | Infrastructure | Repository | membership-repository', ()
 
       it('should update membership attributes', async () => {
```

Comment:
**suggestion,**
Renommer le cas de test existant  comme le nouveau '`'should return the organization and user linked to this membership`): il ne vérifie pas ce qu'il y a en BDD

```suggestion
      it('should return the updated membership`', async () => {
```

Reply:
`      it('should return the updated membership with user and organization`', async () => {`